CHANGELOG.md

```diff
@@ -5,6 +5,133 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 8.11.0
+
+<!-- generated comparing v8.10.2..master -->
+
+_Aug 28, 2025_
+
+We'd like to extend a big thank you to the 19 contributors who made this release possible. Here are some highlights ✨:
+
+- 📊 Add new `SankeyChart`
+- 🚀 Reordering support for row grouping
```

Comment:
Did a minor because of these. Let me know if there anything else you might want to highlight 🎉